From 6d95f7306907e322cf6d41946db3df13242b4dae Mon Sep 17 00:00:00 2001
From: CodeHz <codehz@outlook.com>
Date: Tue, 2 Feb 2021 01:21:46 +0800
Subject: [PATCH 1/6] remove explorer.exe

---
 dlls/user32/win.c | 84 +----------------------------------------------
 1 file changed, 1 insertion(+), 83 deletions(-)

diff --git a/dlls/user32/win.c b/dlls/user32/win.c
index 680defc..37b26e5 100644
--- a/dlls/user32/win.c
+++ b/dlls/user32/win.c
@@ -2065,89 +2065,7 @@ HWND WINAPI FindWindowW( LPCWSTR className, LPCWSTR title )
  */
 HWND WINAPI GetDesktopWindow(void)
 {
-    struct user_thread_info *thread_info = get_user_thread_info();
-
-    if (thread_info->top_window) return thread_info->top_window;
-
-    SERVER_START_REQ( get_desktop_window )
-    {
-        req->force = 0;
-        if (!wine_server_call( req ))
-        {
-            thread_info->top_window = wine_server_ptr_handle( reply->top_window );
-            thread_info->msg_window = wine_server_ptr_handle( reply->msg_window );
-        }
-    }
-    SERVER_END_REQ;
-
-    if (!thread_info->top_window)
-    {
-        STARTUPINFOW si;
-        PROCESS_INFORMATION pi;
-        WCHAR windir[MAX_PATH];
-        WCHAR app[MAX_PATH + ARRAY_SIZE( L"\\explorer.exe" )];
-        WCHAR cmdline[MAX_PATH + ARRAY_SIZE( L"\\explorer.exe /desktop" )];
-        WCHAR desktop[MAX_PATH];
-        void *redir;
-
-        SERVER_START_REQ( set_user_object_info )
-        {
-            req->handle = wine_server_obj_handle( GetThreadDesktop(GetCurrentThreadId()) );
-            req->flags  = SET_USER_OBJECT_GET_FULL_NAME;
-            wine_server_set_reply( req, desktop, sizeof(desktop) - sizeof(WCHAR) );
-            if (!wine_server_call( req ))
-            {
-                size_t size = wine_server_reply_size( reply );
-                desktop[size / sizeof(WCHAR)] = 0;
-                TRACE( "starting explorer for desktop %s\n", debugstr_w(desktop) );
-            }
-            else
-                desktop[0] = 0;
-        }
-        SERVER_END_REQ;
-
-        memset( &si, 0, sizeof(si) );
-        si.cb = sizeof(si);
-        si.lpDesktop = *desktop ? desktop : NULL;
-        si.dwFlags = STARTF_USESTDHANDLES;
-        si.hStdInput  = 0;
-        si.hStdOutput = 0;
-        si.hStdError  = GetStdHandle( STD_ERROR_HANDLE );
-
-        GetSystemDirectoryW( windir, MAX_PATH );
-        lstrcpyW( app, windir );
-        lstrcatW( app, L"\\explorer.exe" );
-        lstrcpyW( cmdline, app );
-        lstrcatW( cmdline, L" /desktop" );
-
-        Wow64DisableWow64FsRedirection( &redir );
-        if (CreateProcessW( app, cmdline, NULL, NULL, FALSE, DETACHED_PROCESS,
-                            NULL, windir, &si, &pi ))
-        {
-            TRACE( "started explorer pid %04x tid %04x\n", pi.dwProcessId, pi.dwThreadId );
-            WaitForInputIdle( pi.hProcess, 10000 );
-            CloseHandle( pi.hThread );
-            CloseHandle( pi.hProcess );
-        }
-        else WARN( "failed to start explorer, err %d\n", GetLastError() );
-        Wow64RevertWow64FsRedirection( redir );
-
-        SERVER_START_REQ( get_desktop_window )
-        {
-            req->force = 1;
-            if (!wine_server_call( req ))
-            {
-                thread_info->top_window = wine_server_ptr_handle( reply->top_window );
-                thread_info->msg_window = wine_server_ptr_handle( reply->msg_window );
-            }
-        }
-        SERVER_END_REQ;
-    }
-
-    if (!thread_info->top_window || !USER_Driver->pCreateDesktopWindow( thread_info->top_window ))
-        ERR( "failed to create desktop window\n" );
-
-    return thread_info->top_window;
+    return 0;
 }
 
 
-- 
2.30.0

