From 8c6fe35c5089d36d130179eb86bdf040795ac21c Mon Sep 17 00:00:00 2001
From: CodeHz <codehz@outlook.com>
Date: Tue, 2 Feb 2021 01:24:10 +0800
Subject: [PATCH 3/6] adjust wine.inf

---
 loader/wine.inf.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 6ae4e96e..cc1bfa6b 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -4047,7 +4047,7 @@ HKLM,Software\Wine\LicenseInformation,"Shell-InBoxGames-SpiderSolitaire-EnableGa
 HKLM,Software\Wine\LicenseInformation,"Shell-PremiumInBoxGames-Chess-EnableGame",0x10001,0x00000001

 [ColorFiles]
-srgb color space profile.icm,"@%11%\mscms.dll,-1"
+; srgb color space profile.icm,"@%11%\mscms.dll,-1"

 [EtcFiles]
 hosts,"@%11%\ws2_32.dll,-1"
@@ -4056,9 +4056,9 @@ protocol,"@%11%\ws2_32.dll,-3"
 services,"@%11%\ws2_32.dll,-4"

 [InfFiles]
-winebus.inf,"@%12%\winebus.sys,-1"
-winehid.inf,"@%12%\winehid.sys,-1"
-wineusb.inf,"@%12%\wineusb.sys,-1"
+; winebus.inf,"@%12%\winebus.sys,-1"
+; winehid.inf,"@%12%\winehid.sys,-1"
+; wineusb.inf,"@%12%\wineusb.sys,-1"

 [NlsFiles]
 c_037.nls
